packages/voter-dapp/src/RetrieveRewards.js

```diff
@@ -225,7 +232,7 @@ function RetrieveRewards({ votingAccount }) {
         })
         .catch(err => console.error(`retrieveRewards.call failed:`, err));
     }
-  }, [votingAccount, rewardsTxn.oldestUnclaimedRound, rewardsTxn.toRetrieve]);
+  }, [web3.utils, drizzle.contracts.Voting, votingAccount, rewardsTxn.oldestUnclaimedRound, rewardsTxn.toRetrieve]);
```

Comment:
Also super strange to see `web3.utils` here

Reply:
yea... i was just following the compiler warnings here, i dont really understand why it wanted this